src/bus/mod.rs

```diff
@@ -0,0 +1,136 @@
+// LNP Node: node running lightning network protocol and generalized lightning
+// channels.
+// Written in 2020 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the MIT License
+// along with this software.
+// If not, see <https://opensource.org/licenses/MIT>.
+
+pub mod ctl;
+pub mod msg;
+pub mod rpc;
```

Comment:
what's your take on renaming this bus to `Info`, as suggested by @TheCharlatan before, either in this PR or in followup?
https://github.com/farcaster-project/farcaster-node/issues/706#issuecomment-1231899553

I think that would make sense, given that you've segregated the cli instructions onto the `Ctl` bus already.

Reply:
I'd prefer doing it in another PR at this stage. I kept `Rpc` because I wasn't convinced about `Info` as I was at the beginning, but `Rpc` is also misleading and I agree we should find another name for it.

Reply:
> I'd prefer doing it in another PR at this stage.

:+1:

> I kept Rpc because I wasn't convinced about Info as I was at the beginning, but Rpc is also misleading and I agree we should find another name for it.

To be clear here: you're not fully satisfied with `Info` as a name either and would rather find another name? My comment https://github.com/farcaster-project/farcaster-node/pull/707#discussion_r986829266 is related to this discussion.

Reply:
To be honest I don't know why I don't like `Info` anymore... so we can go ahead with `Info` :see_no_evil: 

It makes sense to have `Info` for "ready only" and `Ctl` for "mutations", and there is no real difference if it is client/service or service/service. But ACL is easier to implement if "mutations" only happens on `Ctl`.

Reply:
opened #732 to track